examples/restify/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "restify-example",
+  "private": true,
+  "version": "0.18.0",
```

Comment:
Nit: 0.15.0 in core we have 0.18.x

Reply:
Version in examples are not kept in sync in this repo, i think it would be better to do it with lerna instead